bitex/interface/bittrex.py

```diff
@@ -103,13 +102,12 @@ def cancel_order(self, *order_ids, **kwargs):
         return results if len(results) > 1 else results[0]
 
     @format_with(BittrexFormattedResponse)
-    def wallet(self, *args, currency=None, **kwargs):  # pylint: disable=arguments-differ
+    def wallet(self, currency=None, **kwargs):
```

Comment:
See Issue #183 for the reason `*args` and `**kwargs` need to stay.
Moving the pylint comment above the line might actually be prettier.

Reply:
Ok, but *args must stay before currency or changing the order is ok? Because I was assuming, for consistency, that *args and **kwargs ideally should always be the last parameters of the methods. Not put something in the middle like the currency in this case...